manifests/native.json

```diff
@@ -248,6 +248,14 @@
       "mdnPath": "Global_Objects/ArrayBuffer/slice",
       "category": "native"
     },
+    {
+      "type": "native",
+      "moduleName": "buffer-equal",
+      "nodeVersion": "0.11.13",
+      "replacement": "Buffer.prototype.equals",
+      "mdnPath": "Global_Objects/Buffer/equals",
```

Comment:
Wasn't sure what to put for this since it's not on MDN. Is there a nodePath instead?

Reply:
https://nodejs.org/api/buffer.html#bufequalsotherbuffer